src/pipeline/etl/models/templates/model_template.py

```diff
@@ -0,0 +1,43 @@
+"""
+SQLModel Class Template
+---
+
+This module provides a template for creating a new SQLModel class, which
+corresponds to a new table in the database.
+
+To use this template:
+1.  Copy this file to the `src/models/` directory and give it a descriptive name
+    (e.g., `new_data_model.py`).
+2.  Rename the class from `ModelTemplate` to your desired model name (e.g., `NewData`).
+3.  Update the `__tablename__` to match your desired table name in the database.
+4.  Define the fields for your model, replacing the placeholder fields.
+5.  Add the new model to `alembic/env.py` by adding a new import line
+    (e.g., `from pipeline.models.new_data_model import *`).
+"""
+from typing import Optional
+from sqlmodel import Field, SQLModel
+
+
+# TODO: Rename 'ModelTemplate' to a descriptive name for your data model.
+class ModelTemplate(SQLModel, table=True):
```

Comment:
Instead of a ModelTemplate, can this be a BaseModel instead? So that other models can inherit from it. Assuming that there are some attributes that are shared across the models.